_posts/en/newsletters/2019-09-11-newsletter.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Bitcoin Optech Newsletter #63'
+permalink: /en/newsletters/2019/09/11/
+name: 2019-09-11-newsletter
+slug: 2019-09-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a demo implementation of eltoo and
+several discussions related to it, requests comments on limiting the
+normal number of LN gossip updates to once per day, and provides our
+longest-to-date list of notable changes to popular Bitcoin
+infrastructure projects.
+
+{% comment %}<!-- evidence for being "longest-ever":
+    find _posts/en/newsletters/ -type f | while read file ; do echo -n "$file "; sed -n '/^## Notable code/,${/^## [^\(Notable\)]/Q; /linkers\/issues.md/Q; p}' $file | wc -l ; done | sort -n -k2 | tail
+-->{% endcomment %}
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Eltoo sample implementation and discussion:** on the
+ Lightning-Dev mailing list, Richard Myers [posted][eltoo sample] a link
```

Comment:
fwiw interesting eltoo thread launched by Richard Myers an hour ago https://twitter.com/remyers_/status/1171026810005532673?s=20

Reply:
He just sent a longer description to the Lightning-dev mailing list, so I think we're good with just the existing link to that thread in the newsletter.